trin-core/src/portalnet/storage.rs

```diff
@@ -648,14 +645,13 @@ pub mod test {
 
         assert_eq!(32, bytes);
 
-        temp_dir.close()?;
         Ok(())
     }
 
     #[test_log::test(tokio::test)]
     #[serial]
     async fn test_find_farthest_empty_db() -> Result<(), PortalStorageError> {
-        let temp_dir = setup_temp_dir();
```

Comment:
This removal will not keep `temp_dir` in the current scope and `temp_dir` will go out of scope when `setup_temp_dir` finishes.